raiden/tests/integration/network/transport/test_matrix_transport.py

```diff
@@ -542,7 +542,11 @@ def test_monitoring_broadcast_messages(
     transport._client.api.retry_timeout = 0
     transport._send_raw = MagicMock()
     raiden_service = MockRaidenService(None)
-    raiden_service.config = dict(services=ServiceConfig(monitoring_enabled=True))
+    raiden_service.config = RaidenConfig(
+        chain_id=1,
```

Comment:
Can you use the constant for mainnet here? Or is it just any value would be okay here?

Reply:
It's just a random number, I'll randomize it a bit